pkg/crd/manifest.go

```diff
@@ -14,28 +14,63 @@
  * limitations under the License.
  */
 
-package core
+package crd
 
 import (
 	"errors"
 	"fmt"
+	"github.com/ghodss/yaml"
+	"github.com/jinzhu/copier"
+	"github.com/projectriff/riff/pkg/fileutils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	"net/url"
 	"path/filepath"
+	"strings"
+)
 
-	"github.com/projectriff/riff/pkg/fileutils"
+type ResourceChecks struct {
+	Kind     string               `json:"kind,omitempty"`
+	Selector metav1.LabelSelector `json:"selector,omitempty"`
+	JsonPath string               `json:"jsonpath,omitempty"`
+	Pattern  string               `json:"pattern,omitempty"`
+}
 
-	"github.com/ghodss/yaml"
-)
+type Resource struct {
+	Path      string           `json:"path,omitempty"`
+	Content   string           `json:"content,omitempty"`
+	Name      string           `json:"name,omitempty"`
+	Namespace string           `json:"namespace,omitempty"`
+	Checks    []ResourceChecks `json:"checks,omitempty"`
+}
 
-const manifestVersion_0_1 = "0.1"
+type RiffResource struct {
+	System         []Resource `json:"system,omitempty"`
```

Comment:
What is the a distinction between system and initialization?

I'd expect the difference between `riff system install` and `riff namespace init` to be which manifest is applied, not a split within a single manifest.